### PR TITLE
Check if Orca size is more then 1 mb

### LINKED ISSUE #1091: ORCA unit tests can apparently trigger the Ubuntu screen reader, which is also called `orca`
### Environment

N/A

### What is the issue?

This line is the effected issue: https://github.com/Quantum-Accelerators/quacc/blob/93b853b56cf46588e5859bdd4a8b5e0106bb4313/tests/core/recipes/compiled_recipes/test_real_orca_recipes.py#L9

When the user has Ubuntu orca screen reader installed, it tries to run the ORCA test suite because the above logic evalutes to `True` (even when the DFT code ORCA is not installed).

### How can we reproduce the issue?

N/A

### PR DESCRIPTION
**Pull request by rishav-karanjit**
Closes #1091
- [x] I have read the [Developer Guide](https://quantum-accelerators.github.io/quacc/dev/contributing.html). Don't lie! 😉
- [x] I have added relevant unit tests. Note: Your PR will likely not be merged without proper tests.

### CODE REVIEW

File: tests/core/recipes/compiled_recipes/test_real_orca_recipes.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+import os
 
 import pytest
 from ase.build import molecule
 from numpy.testing import assert_allclose
 
 from quacc.recipes.orca.core import relax_job, static_job
 
-has_orca = bool(which("orca"))
+orca_path = which("orca")
```
Comment (Andrew-S-Rosen) [inline tests/core/recipes/compiled_recipes/test_real_orca_recipes.py:10]: Can you change this to `which(SETTINGS.ORCA_CMD)`? This was something I should have done in the first place to not hard-code the value. See [here](https://github.com/Quantum-Accelerators/quacc/blob/722e9dd1cc8b2bc0956aae6e291ca044c103ad71/src/quacc/settings.py#L120-L126) for reference.

File: tests/core/recipes/compiled_recipes/test_real_orca_recipes.py [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+
+if orca_path and os.path.getsize(orca_path) > 1024 * 1024:
```
Comment (Andrew-S-Rosen) [inline tests/core/recipes/compiled_recipes/test_real_orca_recipes.py:12]: @rishav-karanjit: Thanks!! This looks good to me.

One question: do you know the executable size of the Ubuntu orca screen reader? Is it indeed < 1 Mb?

File: tests/core/recipes/compiled_recipes/test_real_orca_recipes.py (hunk 1 above)
Comment (rishav-karanjit) [reply tests/core/recipes/compiled_recipes/test_real_orca_recipes.py]: Thanks. I have a new commit.

File: tests/core/recipes/compiled_recipes/test_real_orca_recipes.py (hunk 2 above)
Comment (rishav-karanjit) [reply tests/core/recipes/compiled_recipes/test_real_orca_recipes.py:12]: Yes. In fact, its in kb.

File: tests/core/recipes/compiled_recipes/test_real_orca_recipes.py (hunk 1 above)
Comment (Andrew-S-Rosen) [reply tests/core/recipes/compiled_recipes/test_real_orca_recipes.py]: Thanks. Looks like you're missing the import statement (`from quacc import SETTINGS`).

Comment (rishav-karanjit) [reply tests/core/recipes/compiled_recipes/test_real_orca_recipes.py]: I also install `deepsource` to my repo. But `deepsource` permission is still failing.

**Comment by Andrew-S-Rosen** [comment]
@rishav-karanjit: Thanks. Please address the [`DeepSource` message](https://app.deepsource.com/gh/Quantum-Accelerators/quacc/run/e1bd45ad-8b7f-4871-809e-7272990a3efe/python/) and then this is good to merge once all checks appear.

**Comment by rishav-karanjit** [comment]
> @rishav-karanjit: Thanks. Please address the [`DeepSource` message](https://app.deepsource.com/gh/Quantum-Accelerators/quacc/run/e1bd45ad-8b7f-4871-809e-7272990a3efe/python/) and then this is good to merge once all checks appear.

I also install deepsource to my repo. But deepsource permission is still failing.

**Comment by Andrew-S-Rosen** [comment]
@rishav-karanjit: You don't need to install deepsource. You just need to address the message linked above and that you can see when you click "Details"

**Comment by Andrew-S-Rosen** [comment]
Thank you for your contribution! I have set this to auto-merge once tests pass.

**Pull request merged**